src/Router.js

```diff
@@ -1,17 +1,102 @@
+import ErrorState from '@redhat-cloud-services/frontend-components/ErrorState';
 import { useFlag } from '@unleash/proxy-client-react';
+import PropTypes from 'prop-types';
 import { Route, Routes } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
 
 import EdgeImageDetail from './Components/edge/ImageDetails';
 import ShareImageModal from './Components/ShareImageModal/ShareImageModal';
+import { useGetComposesQuery } from './store/imageBuilderApi';
 import { manageEdgeImagesUrlName } from './Utilities/edge';
+import { resolveRelPath } from './Utilities/path';
 
 const LandingPage = lazy(() => import('./Components/LandingPage/LandingPage'));
 const CreateImageWizard = lazy(() =>
   import('./Components/CreateImageWizard/CreateImageWizard')
 );
 
+/**
+ * Help page for newcomers creating their first image if they've never did it.
+ * The user gets the possibility to click on a `create image` button that will
+ * open the wizard for them.
+ *
+ * caveat:
+ * This component takes care of displaying itself the CreateImageWizard. Meaning
+ * that the wizard is not being displayed using the navigation system. This is
+ * valid for-now™ [2023-09] because the wizard is being displayed in a modal.
+ * This would most likely have to get updated when the wizard will get changed
+ * to an InPage mode.
+ */
+const AppZeroState = () => {
+  const [wizardOpenned, setWizardOpenned] = useState(false);
+
+  return (
+    <AsyncComponent
+      appName="dashboard"
+      module="./AppZeroState"
+      scope="dashboard"
+      ErrorComponent={<ErrorState />}
+      app="Images"
+      appId="images_zero_state"
+      customText="Create your image by using the Image Builder wizard."
+      customButton={
+        <>
+          {wizardOpenned && <CreateImageWizard />}
+          <Button
+            className="pf-c-button pf-m-primary pf-u-p-md pf-u-font-size-md"
+            data-testid="create-image-action"
+            onClick={() => setWizardOpenned(true)}
+          >
+            Create image
+          </Button>
+        </>
+      }
+    />
+  );
+};
+
+const ZERO_IMAGES = 0;
+
+/**
+ * Navigate between the LandinPage and the ZeroState depending on is the user is
+ * a newcomer of no. For now, newcomer are defined by the amount of composes
+ * they have in the Image Builder database (0 Meaning newcomer).
+ *
+ * @param from is a protection against infinite navigate loops. Values can be
+ * 'ZeroState' or 'LandinPage'.
+ */
+const ZeroStateNavigation = ({ from }) => {
+  const navigate = useNavigate();
+  const { data, isSuccess } = useGetComposesQuery({
+    limit: 10,
+  });
+  if (!isSuccess) {
+    return undefined;
+  }
+  const composes = data.data;
+  switch (from) {
+    case 'ZeroState':
+      // navigating to the LandinPage is only allowed from the ZeroState page
+      if (composes.length > ZERO_IMAGES) {
+        navigate(resolveRelPath(''));
+      }
+      break;
+    case 'LandinPage':
```

Comment:
Small typo here and in a few of the comments, this should be LandingPage (with a g :wink:).